.buildkite/pipeline.yml

```
@@ -31,28 +31,36 @@ steps:
   - wait
 
   - label: "Run JRA55 tests"
-    key: "tests_jra55"
+    key: "test_jra55"
     env:
       CUDA_VISIBLE_DEVICES: "-1"
       TEST_GROUP: "jra55"
     commands:
       - "julia --project -e 'using Pkg; Pkg.test()'"
 
   - label: "Run ECCO tests"
-    key: "tests_ecco"
+    key: "test_ecco"
     env:
       CUDA_VISIBLE_DEVICES: "-1"
       TEST_GROUP: "ecco"
     commands:
       - "julia --project -e 'using Pkg; Pkg.test()'"
 
   - label: "Run surface flux tests"
-    key: "tests_fluxes"
+    key: "test_fluxes"
     env:
       CUDA_VISIBLE_DEVICES: "-1"
       TEST_GROUP: "fluxes"
     commands:
       - "julia --project -e 'using Pkg; Pkg.test()'"
 
+  - label: "Run simulation tests"
+    key: "test_simulations"
+    env:
+      TEST_GROUP: "simulations"
+    commands:
+      - "julia --project -e 'using Pkg; Pkg.test()'"
+
```

I guess you need to append something like this

```suggestion
    agents:
      slurm_mem: 120G
      slurm_gpus: 1
      slurm_cpus_per_task: 8
      slurm_ntasks: 1
      slurm_gpus_per_task: 1
      slurm_time: 01:00:00
```